### DISCUSSION TITLE
No rescaling

### DISCUSSION THREAD

**Discussion by P-e-a-s-h-o-o-t-e-r** [xinntao/Real-ESRGAN#451] [Ideas]
Please include an upscale ratio of 1, where there will be no upscaling but just the removal of compression artifacts and a sharpening of the image.

**Comment by Deadstarrr** [comment]
Absolutely. Why does this force an upscale? I don't want that.

**Comment by haobo724** [comment]
actually you can adjust the ratio by argument "-s" in python inference , no scale just remove the bluring or artifacts is possible
